content/docs/introduction/install.md

```diff
@@ -85,3 +85,20 @@ docker run -p 9090:9090 my-prometheus
 
 A more advanced option is to render the config dynamically on start
 with some tooling or even have a daemon update it periodically.
+
+## Using Configuration Management Systems
+
+If you prefer using configuration management systems you might be interested in the following user contributions:
```

Comment:
Maybe let's say "third-party" instead of "user". That's consistent with our terminology for exporters and client libraries.